.github/workflows/build-test.yml

```
@@ -54,6 +54,9 @@ jobs:
     - name: Package Manager Test
       run: go test -run TestPackages -v ./...
 
+    - name: MacOS Native Signing Test
+      run: go test -run TestMacOSSigning -v ./...
+
```

We should only need to change the test from `TestCherriNoSign` to just `TestCherri`. We could add sign to the end of the job/task to denote it does this

```suggestion
build-test-macos-sign:
runs-on: macos-latest
steps:
- uses: actions/checkout@v6

- name: Set up Go
uses: actions/setup-go@v3
with:
go-version: 1.24

- name: Build
run: go build -v ./...

- name: Test
run: go test -run TestCherri -v ./...

- name: Decompilation Test
run: go test -run TestDecomp -v ./...

- name: Package Manager Test
run: go test -run TestPackages -v ./...

```